src/rabbit_plugins.erl

```diff
@@ -123,6 +127,24 @@ dependencies(Reverse, Sources, AllPlugins) ->
     true = digraph:delete(G),
     Dests.
 
+%% Filter real plugins from application dependencies
+is_strict_plugin(#plugin{extra_dependencies = ExtraDeps}) ->
```

Comment:
I'd suggest renaming this to either `is_strictly_plugin/1` or `is_actual_plugin/1`.